dvc/repo/get.py

```diff
@@ -20,13 +21,27 @@ def __init__(self):
         )
 
 
+def resolve_artifact_addressing(path):  # add tests
```

Comment:
```suggestion
def resolve_artifact_path(path):
```
Is this always a file path? "path" is consistent with the variable name returned
if it's more generic it should be "url" or "address" I think - consistently though 😄